src/webgpu/api/validation/gpu_external_texture_expiration.spec.ts

```diff
@@ -322,42 +322,3 @@ g.test('import_from_different_video_frame')
       t.submitCommandBuffer(bindGroup, true);
     });
   });
-
-g.test('device_destroyed_before_importing_video_frame')
-  .desc(
-    `
-    Tests that imported GPUExternalTexture from destroyed device and submit
-    it should result in an error.
-    `
-  )
-  .params(u =>
-    u //
-      .combine('sourceType', ['VideoElement', 'VideoFrame'] as const)
-  )
-  .fn(async t => {
-    const sourceType = t.params.sourceType;
-    const videoElement = t.getDefaultVideoElementAndCheck();
-
-    let bindGroup: GPUBindGroup;
-    let externalTexture: GPUExternalTexture;
-    let source: HTMLVideoElement | VideoFrame;
-    await startPlayingAndWaitForVideo(videoElement, async () => {
-      source =
-        sourceType === 'VideoFrame'
-          ? await getVideoFrameFromVideoElement(t, videoElement)
-          : videoElement;
-
-      t.device.destroy();
```

Comment:
If we keep this test, perhaps add something like:
`if (t.params.awaitAfterDestroy) await t.device.lost;`
here.

Reply:
Move test case to api,validation,state,device_lost,destroy